website/content/en/preview/tasks/pod-density.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control Pod Density"
```

Comment:
```suggestion
title: "Pod Density"
```

Reply:
Control sends the wrong message, I think. Ultimately karpenter is going to have the say here. Limit might be more accurate, but I'd probably just drop any verb from the title.

Reply:
I think of pages under the task section fitting into a "I want to X" pattern. perhaps control is implied?